src/main/java/org/myrobotlab/mqtt/MqttMsg.java

```diff
@@ -0,0 +1,147 @@
+package org.myrobotlab.mqtt;
+
+public class MqttMsg {
```

Comment:
for pojo's it's a good idea to have a hashCode / equals / toString operators.

Reply:
fixed